src/App.ts

```diff
@@ -0,0 +1,67 @@
+/**
+ * App
+ * Define a Restful Webservice and routes controller requests
+ */
+
+import * as bodyParser from "body-parser";
+import * as express from "express";
+import * as core from "express-serve-static-core";
+import * as http from "http";
+import { CONFIG } from "./Configuration";
+import { NotificationController } from "./controllers/NotificationController";
+import { logger } from "./utils/Logger";
+
+// Define server and routes
+export class App {
+  public static startApp(): http.Server {
```

Comment:
since this class has no state (no members), please convert all these static methods to simple functions.
as a general advice, use simple functions instead of a class with only static methods.